docs/docs/building-and-running.md

```diff
@@ -82,8 +82,8 @@ Now you can build the docker containers locally - this might take quite a while
 
 ```bash
 eval $(minikube docker-env) # (If you haven't run this already)
-npx lerna run build --stream --concurrency=1 --include-dependencies # (if you haven't run this already)
-npx lerna run docker-build-local --stream --concurrency=1 --include-filtered-dependencies
+yarn lerna run build --stream --concurrency=1 --include-dependencies # (if you haven't run this already)
+yarn lerna run docker-build-local --stream --concurrency=1 --include-filtered-dependencies
```

Comment:
`--include-filtered-dependencies` is to be used for lerna ^3.18.0
```suggestion
yarn lerna run build --stream --concurrency=1 --include-filtered-dependencies # (if you haven't run this already)
yarn lerna run docker-build-local --stream --concurrency=1 --include-filtered-dependencies
# If you are using lerna 3.18.0 or higher, run these instead:
yarn lerna run build --stream --concurrency=1 --include-dependencies # (if you haven't run this already)
yarn lerna run docker-build-local --stream --concurrency=1 --include-dependencies
```

Reply:
lerna 3.18.0 has been released 6 months ago.
How about put version requirement in `Prerequisites` section on the top to avoid making our document even more complex to follow?
@AlexGilleran any ideas?

Reply:
I think that's a great idea :+1:

Reply:
:+1: :+1:

Reply:
Better still would be to just remove lerna as a dependency from the docs completely, and make it so every `lerna` command is through `yarn` and uses the version installed by `yarn` (which I think we might have already done in this PR?)